packages/react-examples/src/react/DetailsList/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -46,35 +49,64 @@ const classNames = mergeStyleSets({
 const ITEMS_PER_GROUP = 20;
 const GROUP_COUNT = 20;
 
-export class DetailsListCustomGroupHeadersExample extends React.Component<{}, {}> {
+export class DetailsListCustomGroupHeadersExample extends React.Component<{}, { selectedItem: IDropdownOption }> {
   private _items: IExampleItem[];
   private _groups: IGroup[];
+  private dropdownStyles: Partial<IDropdownStyles> = { dropdown: { width: 300 } };
+  private dropdownControlledExampleOptions = [
```

Comment:
These should probably be moved to the top of the file along with the other constants. Also, it would be better to call the second one just `dropdownOptions`.

Reply:
Done!